src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2190,6 +2190,38 @@ private function addMailerSection(ArrayNodeDefinition $rootNode, callable $enabl
                             ->end()
                         ->end()
                     ->end()
+                    ->children()
+                        ->arrayNode('dkim_signer')
+                            ->fixXmlConfig('option')
+                            ->canBeEnabled()
+                            ->info('DKIM signer configuration')
+                                ->children()
```

Comment:
the indentation is odd, children() shouldn't be nested under info() (same below)